src/amuse/community/phiGRAPE/Makefile

```diff
@@ -24,35 +36,58 @@ CODELIB=$(BUILDDIR)/libphigrape.a
 
 GLOBJECTS=src/fthread.o src/viewer.o src/start_viewer.o src/libpt.a
 
-CUDA_SDK ?= /disks/koppoel1/CUDA23/cudasdk/C
-CUDA_TK  ?= /disks/koppoel1/CUDA23/cuda
+ifeq ($(strip $(CUDA_SDK)),)
+    CUDA_SDK = /disks/koppoel1/CUDA23/cudasdk/C
+endif
+ifeq ($(strip $(CUDA_TK)),)
+    CUDA_TK = /disks/koppoel1/CUDA23/cuda
+endif
 
-CUDA_LIBDIRS ?= -L$(CUDA_TK)/lib -L$(CUDA_TK)/lib64
-CUDA_LIBS ?= -lcudart
+ifeq ($(strip $(CUDA_LIBDIRS)),)
+    CUDA_LIBDIRS = -L$(CUDA_TK)/lib -L$(CUDA_TK)/lib64
+endif
+ifeq ($(strip $(CUDA_LIBS)),)
+    CUDA_LIBS = -lcudart
+endif
 
-SAPPORO_LIBDIRS ?= -L$(AMUSE_DIR)/lib/sapporo_light
-OPENMP_CFLAGS ?= 
-SAPPORO_LIBS ?= $(SAPPORO_LIBDIRS) -lsapporo $(OPENMP_CFLAGS)
-GRAPE6_LIBDIRS ?= -L$(GRAPE6)/lib
+ifeq ($(strip $(SAPPORO_LIBDIRS)),)
+    SAPPORO_LIBDIRS = -L$(AMUSE_DIR)/lib/sapporo_light
+endif
+ifeq ($(strip $(OPENMP_CFLAGS)),)
+    OPENMP_CFLAGS = SAPPORO_LIBS ?= $(SAPPORO_LIBDIRS) -lsapporo $(OPENMP_CFLAGS)
```

Comment:
this doesn't look right...

Reply:
Yes I see, clearly not right! (also original had
 OPENMP_CFLAGS ?=
So optional not setting, which is very strange)

I'll update this one